src/reynir_correct/checker.py

```diff
@@ -247,10 +247,15 @@ def annotate(self, sent: Sentence) -> List[Annotation]:
                     assert original_terminal is not None
                     terminal = grammar.terminals[original_terminal]
                     assert isinstance(terminal, VariantHandler)
-                    if t.suggestion_does_not_match(terminal, bin_token):
-                        # If this token is annotated with a spelling suggestion,
-                        # do not add it unless it works grammatically
+                    try:
+                        if t.suggestion_does_not_match(terminal, bin_token):
+                            # If this token is annotated with a spelling suggestion,
+                            # do not add it unless it works grammatically
+                            annotate = False
+                    except AttributeError:
+                        # 1,8 milljarður kóna - wrap_tokens seems to fail
```

Comment:
We need to dig further into this. Could be that TOK.Amount tokens are not converted to CorrectToken instances. (And that might be fixed in the newest master.)

Reply:
Changed this back, will look into this and test with newest master.

Reply:
Changed this back. Will look into better.